packages/dev/README.md

```diff
@@ -0,0 +1,70 @@
+# `clerk-dev` CLI
+
+## Installation
+
+```sh
+git clone --single-branch --branch ds.feat/clerk-dev-cli https://github.com/clerk/javascript clerk-dev-cli
+cd clerk-dev-cli
+npm install
+npm install --global ./packages/dev
+```
```

Comment:
This is a great question! Initially I was thinking it could be a local-only package, and just always source from your current checkout of the monorepo. This would alleviate the need to version and publish the package, and ensure that you always have the most up-to-date version. However, this does also assume that the branch you're on is up-to-date with `main`, which I realized might not always be the case. The CLI is only intended for use with the `clerk/javascript` monorepo, so I wasn't really sure that it warranted having the full publishing workflow.

That being said, this is a loose opinion weakly held, so if you feel strongly that it should be published to npm I'm happy to make the changes necessary to `package.json` to make that happen!

Reply:
In the past I used such a dev-cli not only on my personal computer but also instructed users/customers to run it for me (debugging on a call). Or outside contributors used it for their PR. I've also used it in CI.
In all those cases it was helpful or even essential that it was published on npm.

Since we already have our publishing setup done and it's easy to publish, I'd vote for doing that with this package, too!

Reply:
Updated in [87b81da](https://github.com/clerk/javascript/pull/3689/commits/87b81da0d7ee463e6aba07a67baa74214219f50b)